frontend/src/components/DataCard/SuggestedLabels.jsx

```diff
@@ -1,15 +1,21 @@
 import React from "react";
+import * as stringSimilarity from "string-similarity";
 
 export default function SuggestedLabels({ card, labels }) {
     if (labels.length <= 5) {
         return null;
     }
 
+    const labelStrings = labels.map(label => `${label.name} (${label.description})`);
+
+    let matches = stringSimilarity.findBestMatch(card.text.text, labelStrings);
+    matches = matches.ratings.sort((rating1, rating2) => parseFloat(rating2.rating) - parseFloat(rating1.rating)).slice(0, 5);
```

Comment:
Would you mind adding a comment here to explain how this works?

Reply:
Sure!

`labelStrings` is going to be an array of strings with the format: "{label name} {label description}"

The `stringSimilarity.findBestMatch` takes a string and an array of strings to preform the similarity comparisons and is saved in the variable `matches`.

`matches` is then sorted from best rating to lowest rating (ratings are floats, so they're parsed first then compared against each other.) 

Then I'm simply splicing to only get the first 5 results to loop through in the JSX.

Reply:
Lol I meant please add that explanation to the code in the form of a comment so we can know what it's doing later :)